index.php

```diff
@@ -21,64 +21,75 @@
 
 <!DOCTYPE html>
 <html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
-    <head>
-        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
-        <meta charset="utf-8">
-
-        <title>AKAI 2020</title>
-
-        <!-- Fonts -->
-        <link href="https://fonts.googleapis.com/css?family=Nunito:200,600" rel="stylesheet">
-        <style>
-        .wideField
-            {
-                width: 400px;
-            }
-        </style>
-    </head>
-    <body>
-        
-         <nav class="navbar navbar-expand-md navbar-dark bg-dark fixed-top">
-      <pre class="navbar-brand">AKAI 2020 - wniosek o stypendium</pre>
-      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarsExampleDefault" aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
-        <span class="navbar-toggler-icon"></span>
-      </button>
 
+<head>
+    <meta charset="utf-8">
+
+    <title>AKAI 2020</title>
+
+    <!-- Fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Nunito:200,600" rel="stylesheet">
+    <link href="style/style.css" rel="stylesheet">
+</head>
+
+<body>
+
+    <nav class="navbar">
+        <header class="navbar-brand">AKAI 2020 - wniosek o stypendium</header>
```

Comment:
To trzeba zmienić na bardziej ogólne w stylu AKAI Generator Dokumentów - chcemy to udostępnić na zewnątrz przecież.